src/Response/ArraySerializer.php

```diff
@@ -23,6 +23,14 @@ final class ArraySerializer
 {
     /**
      * Serialize a response message to an array.
+     *
+     * @return array{
```

Comment:
Do we use this kind of array somewhere?
If so, I would prefer having a psalm-type so we can import that array shape for further usage.

Reply:
We don't currently use this